ui/app/templates/components/clients/history.hbs

```diff
@@ -1,213 +1,205 @@
-{{#if (and (eq @tab "history") (eq @model.config.queriesAvailable false))}}
-  {{#if (eq @model.config.enabled "On")}}
-    <EmptyState
-      @title="No monthly history"
-      @message="There is no data in the monthly history yet. We collect it at the end of each month, so your data will be available on the first of next month."
-    />
-  {{else}}
-    <EmptyState
-      @title="Data tracking is disabled"
-      @message="Tracking is disabled, and no data is being collected. To turn it on, edit the configuration."
-    >
-      {{#if @model.config.configPath.canUpdate}}
-        <p>
-          <LinkTo @route="vault.cluster.clients.index" @query={{hash tab="config"}}>
-            Go to configuration
-          </LinkTo>
-        </p>
-      {{/if}}
-    </EmptyState>
-  {{/if}}
-{{else}}
-  <div class="box is-sideless is-fullwidth is-marginless is-bottomless">
-    {{#if (eq @tab "current")}}
-      <h1 data-test-client-count-title class="title is-4 has-bottom-margin-s">
-        Current month
-      </h1>
-      <p class="has-bottom-margin-s">
-        The below data is for the current month starting from the first day. For historical data, see the monthly history
-        tab.
-      </p>
-      {{#if (eq @model.config.enabled "Off")}}
-        <EmptyState
-          @title="Tracking is disabled"
-          @message="Tracking is disabled and data is not being collected. To turn it on edit the configuration."
-        >
-          {{#if @model.config.configPath.canUpdate}}
-            <LinkTo @route="vault.cluster.clients.edit">
+<div class="box is-sideless is-fullwidth is-marginless is-bottomless">
```

Comment:
Will want to move all static info to the the history route, and only have dynamic stuff reliant on API response and getters in component - ya?